### DISCUSSION TITLE
database

### DISCUSSION THREAD

**Discussion by kb-nett** [quasarframework/quasar#13638] [General - Components / Directives / etc] [answered]
Hi, I am new to quasar. I want to change a web app i wrote using Django and PostGres to Quasar framework but unable to find documentation for database connectivity using quasar. Kindly tell me if there is any docs available for how to do this?  

Thanks in advance for your help!

**Comment by MilosPaunovic** [comment]
What do you mean by database connectivity? Do you have some kind of backend communication with the DB?

**Comment by suleimanas** [comment, marked as answer]
Quasar is your frontend part of the app you'll not connect to the database from here.

**Reply by kb-nett** [reply]
Ok, I got it. I shall now convert my django app to REST API and connect to it from quasar. Thanks for the guidance. I was bogged down into finding a direct connectivity to DB.
